omegalambda/main/common/util/filereader_utils.py

```diff
@@ -129,8 +136,59 @@ def gaussianfit(x, a, x0, sigma):
     return a*np.exp(-(x-x0)**2/(2*sigma**2))
 
 
-def radial_average(path: str, saturation: Union[int, float]) -> Tuple[Optional[Union[float, int]],
-                                                                      Union[float, int], bool]:
+@jit(parallel=True, nogil=True)
+def _get_all_fwhm(stars, peaks, data, ri, sky, binsize):
```

Comment:
Reminder to keep up with frequent commenting.  Code becomes hard to read and understand without it.

Reply:
The method name describes what it does quite well but I'm more interested in the data types that are passed.